libs/libc/uuid/lib_uuid_stream.c

```diff
@@ -0,0 +1,181 @@
+/****************************************************************************
+ * libs/libc/uuid/lib_uuid_stream.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <sys/types.h>
+#include <endian.h>
+#include <uuid.h>
+
+/****************************************************************************
```

Comment:
Did this code come from NetBSD?  It looks a lot like the code here
http://mirrors.mit.edu/NetBSD/NetBSD-release-7/src/lib/libc/uuid/uuid_stream.c

If so my understanding is that we would leave the BSD headers on this and call it out in the LICENSE file

Reply:
The code is rewritten from scratch now. Should we put some note in LICENSE file?

Reply:
I think this is fine now.  Most of these functions are trivial especially with us using our existing endian code.